webui/src/pages/repositories/services/import_data.jsx

```diff
@@ -186,7 +186,7 @@ const ImportForm = ({
             }
             <Form.Group className='form-group'>
                 <Form.Label><strong>Commit Message:</strong></Form.Label>
-                <Form.Control sm={8} type="text" ref={commitMsgRef} name="text" autoFocus/>
+                <Form.Control sm={8} type="text" ref={commitMsgRef} name="text" autoFocus placeholder={'Import data from...'}/>
```

Comment:
Make it a default value (rather than a placeholder), and use something like: "Imported data from S3".